PowerEditor/installer/nativeLang/korean.xml

```diff
@@ -971,17 +1045,17 @@
   <Item id="7004" name="탭 정렬(&amp;t)"/> 
 </Window>
 <ColumnEditor title="열 편집기">
-  <Item id="2023" name="글자 삽입"/> 
-  <Item id="2033" name="숫자 삽입"/> 
-  <Item id="2030" name="초기 숫자:"/> 
-  <Item id="2031" name="증가치:"/> 
-  <Item id="2035" name="숫자 앞에 0 채우기"/> 
-  <Item id="2036" name="반복:"/>
-  <Item id="2032" name="형식"/> 
-  <Item id="2024" name="10진수"/> 
-  <Item id="2025" name="8진수"/> 
-  <Item id="2026" name="16진수"/> 
-  <Item id="2027" name="2진수"/> 
+  <Item id="2023" name="글자 삽입(&amp;T)"/>
+  <Item id="2033" name="숫자 삽입(&amp;N)"/>
+  <Item id="2030" name="초기 숫자(&amp;I):"/>
+  <Item id="2031" name="증가치(&amp;y):"/>
+  <Item id="2035" name="숫자 앞에 0 채우기(&amp;z)"/>
+  <Item id="2036" name="반복(&amp;R):"/>
+  <Item id="2032" name="형식(&amp;F)"/>
```

Comment:
Sorry I just fix an issue about it:
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
&amp;F should be removed.